examples/el_farol/agent.py

```diff
@@ -0,0 +1,71 @@
+from mesa import Agent
```

Comment:
There are multiple agent classes defined in this file, so the filename should be `agents.py`.